app/api/user_api/v1/sessions.rb

```diff
@@ -21,50 +15,16 @@ def create_access_token(expires_in:, account:, application:)
           requires :email
           requires :password
           requires :application_id
+          optional :remember_me, type: Boolean
           optional :expires_in, allow_blank: false
           optional :otp_code, type: String,
                               desc: 'Code from Google Authenticator'
         end
 
         post do
-          declared_params = declared(params, include_missing: false)
-          account = Account.kept.find_by(email: declared_params[:email])
-          error!('Invalid Email or Password', 401) unless account
-
-          application = Doorkeeper::Application.find_by(uid: declared_params[:application_id])
-          error!('Wrong Application ID', 401) unless application
-          error!('Your account was locked!', 401) unless account.locked_at.nil?
-
-          unless account.valid_password? declared_params[:password]
-            account.add_failed_attempt
-            error!('Invalid Email or Password', 401)
-          end
-
-          unless account.active_for_authentication?
-            error!('You have to confirm your email address before continuing', 401)
-          end
-
-          unless account.otp_enabled
-            account.refresh_failed_attempts
-            return create_access_token expires_in: declared_params[:expires_in],
-                                       account: account,
-                                       application: application
-          end
-
-          if declared_params[:otp_code].blank?
-            account.add_failed_attempt
-            error!('The account has enabled 2FA but OTP code is missing', 403)
-          end
-
-          unless Vault::TOTP.validate?(account.uid, declared_params[:otp_code])
-            account.add_failed_attempt
-            error!('OTP code is invalid', 403)
-          end
-
-          account.refresh_failed_attempts
-          create_access_token expires_in: declared_params[:expires_in],
-                              account: account,
-                              application: application
+          ::Services::AuthService.sign_in(params: declared(params),
+                                          device_params: env['device_params'],
+                                          device_uuid: env['device_uuid'])
```

Comment:
Previously this endpoint returned just the jwt, now it's returning `{"token": "..."}`
Could this again just send the jwt for consistency?  Any client using this api would otherwise need updating.